packages/api/src/modules/providers/demarchesSimplifiees/schemas/schema-72319-form.json

```diff
@@ -0,0 +1,95 @@
+{
+  "demarcheId": 72319,
+  "schema": [
+    {
+      "to": "service_instructeur",
+      "from": "service.organisme"
+    },
+    {
+      "from": "siret",
+      "to": "siret"
+    },
+    {
+      "from": "demande.state",
+      "to": "status"
+    },
+    {
+      "from": "demande.champs.Q2hhbXAtMzYzNDkz.value",
+      "to": "actions_proposee[0].intitule"
+    },
+    {
+      "from": "demande.champs.Q2hhbXAtMjcxNjY1Ng==.value",
+      "to": "actions_proposee[0].objectifs"
+    },
+    {
+      "from": "demande.champs.Q2hhbXAtMzYzNDk3.value",
+      "to": "actions_proposee[0].description"
+    },
+    {
+      "from": "demande.champs.Q2hhbXAtMzEzMzI0MQ==.value",
+      "to": "montants.demande"
+    },
+    {
+      "from": "demande.champs.Q2hhbXAtMzYzNTIz.value",
+      "to": "montants.total"
+    },
+    {
+      "from": "demande.annotations.Q2hhbXAtMjcxNzMwNA==.value",
+      "to": "montants.accorde"
+    },
+    {
+      "from": "demande.annotations.Q2hhbXAtMjY3NDMxMA==.value",
+      "to": "ej"
+    },
+    {
+      "from": "demande.annotations.Q2hhbXAtMjY3NDMxMA==.value",
+      "to": "versementKey"
+    },
+    {
+      "from": "demande.annotations.Q2hhbXAtMjY3NDI3Ng==.value",
+      "to": "date_commision"
+    },
+    {
+      "from": "demande.dateDepot",
+      "to": "transmis_le"
+    },
+    {
+      "value": "Programme 177 « Hébergement, parcours vers le logement et insertion des personnes vulnérables »",
+      "to": "dispositif"
+    }
+  ],
+  "commonSchema": [
+    {
+      "to": "siret",
+      "from": "siret"
+    },
+    {
+      "to": "service_instructeur",
+      "from": "service.organisme"
+    },
+    {
+      "to": "dispositif",
+      "value": "Programme 177 « Hébergement, parcours vers le logement et insertion des personnes vulnérables »"
+    },
+    {
+      "to": "montants.accorde",
+      "from": "demande.annotations.Q2hhbXAtMjcxNzMwNA==.value"
+    },
+    {
+      "to": "montants.demande",
+      "from": "demande.champs.Q2hhbXAtMzEzMzI0MQ==.value"
+    },
+    {
+      "to": "objet",
+      "from": "demande.champs.Q2hhbXAtMzYzNDkz.value"
+    },
+    {
+      "to": "providerStatus",
+      "from": "demande.state"
+    },
+    {
+      "to": "dateTransmitted",
```

Comment:
la diff "to" -> "from" et "to" -> "value" c'est du graphQL ?

Reply:
Non c'est la spec de notre format. Victor prévoyait que le champ DS indiqué par `from` soit mappé vers le champ indiqué par `to` dans notre format. J'ai rajouté la possibilité de renseigner `value` au lieu de `from` pour renseigner une valeur statique sur un champ plutôt que de le mapper à un champ de la démarche DS. C'est clair ou non ?